src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -31,9 +32,18 @@ internal sealed class ProcessMetrics
     private double? virtualMemoryUsage;
     private double? userProcessorTime;
     private double? privilegedProcessorTime;
+    private int? numberOfThreads;
+    private IEnumerable<Measurement<double>> cpuUtilization;
+
+    // vars for calculating CPU utilization
+    private DateTime collectionTimeStamp;
```

Comment:
nit: You could rename these variables to something more intuitive such as `lastCollectionTimeStamp`, `lastCollectedUserProcessTime` etc.

Reply:
renamed the vars as suggested; thank you

Reply:
I think it'd be better to rename the two process time variables to have the prefix "lastCollected" instead of "lastCollection".

Reply:
yeah, that's more succinct